src/Years/index.js

```diff
@@ -126,11 +126,16 @@ export default class Years extends Component {
     );
   }
 
+  componentWillMount() {
```

Comment:
Why are we doing this in `componentWillMount`? This should go either in the render function or the constructor.

FWIW, all `componentWill` methods are deprecated in latest react.

Reply:
Updated